Bottom: e158cfb023f019d4ef91f72e39f049c35cf5993e
Top:    1e3526d39b2591bb10a08bfed2622f05f82b1bd8
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-11 10:49:19 -0700

Refresh of rstream-delay

---

diff --git a/examples/rstream.c b/examples/rstream.c
index a52a262..7f39b29 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -56,7 +56,7 @@ static int test_size[] = {
 static int use_rs = 1;
 static int use_async;
 static int verify;
-static int flags;
+static int flags = MSG_DONTWAIT;
 static int no_delay = 1;
 static int custom;
 static int iterations = 1;
